### ISSUE TITLE
Online install method over mobile data

### ISSUE THREAD

**Issue by Nemoswrold** [khcrysalis/Feather#78]
I’m not sure if you’re aware, but the online installation works perfectly over WiFi. However, it doesn’t seem to work on mobile data. If you’re already aware of this, feel free to disregard.

**Comment by kitsah** [comment]
Yes, this is correct. Online installation method does not work via cellular.

**Comment by khcrysalis** [comment]
we're working on it, its currently being tested

**Comment by Cristiandis** [comment]
does it really not work?
it works for me

**Comment by Cristiandis** [comment]
https://github.com/user-attachments/assets/3183228d-d523-435d-9324-edde79191b97

**Comment by GottaLoveAng** [comment]
it was recently fixed that's why. some people are still having issues but it's possibly just isp blocking it rather than the actual feature being broken now.

**Comment by Cristiandis** [comment]
> it was recently fixed that's why. some people are still having issues but it's possibly just isp blocking it rather than the actual feature being broken now.

UPDATE: i bought a proper cert and now it doesn't work, seems to only work with revoked enterprise certs or smth

**Comment by khcrysalis** [comment]
it seems to be related to entitlements, some could work and some could not.

**Comment by Cristiandis** [comment]
> it seems to be related to entitlements, some could work and some could not.

esign works over cellular data if you enable hotspot

**Comment by khcrysalis** [comment]
Nothing new

**Comment by Cristiandis** [comment]
does this mean the dev cert needs a specific entitlement, or does it just happen because of incompatible entitlements?

**Comment by TheDuckMan64** [comment]
> it was recently fixed that's why. some people are still having issues but it's possibly just isp blocking it rather than the actual feature being broken now.

Any reason you don't hardcode the local server address to localhost? Seems to fix signing over cellular.

**Comment by turbopug2735** [comment]
any progress on this issue?
